shopinvader_utm/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . import services
+from . import controllers
+from . import tests
```

Comment:
`tests` should never be imported. It's automatically done when you're in test-mode.